lib/msal-browser/src/broker/client/EmbeddedClientApplication.ts

```diff
@@ -166,14 +167,15 @@ export class EmbeddedClientApplication {
                     if (brokerHandshakeResponse) {
                         clearTimeout(timeoutId);
                         this.logger.info(`Received handshake response: ${JSON.stringify(brokerHandshakeResponse)}`);
+                        window.removeEventListener("message", onHandshakeResponse);
                         resolve(brokerHandshakeResponse);
                     } else {
-                        this.logger.warning(`Message is not handshake response: ${message}`);
+                        this.logger.warning(`Message is not handshake response: ${JSON.stringify(message.data)}`);
```

Comment:
I would split this into two messages, with the data included in a verbose message.

Reply:
Also, may contain PII?

Reply:
done.